testng-eclipse-plugin/src/main/org/testng/eclipse/ui/util/ConfigurationHelper.java

```diff
@@ -407,7 +410,12 @@ else if (type == LaunchType.PACKAGE) {
         IType t = ijp.findType(entry.getKey());
         if (t != null) {
           for (String mthd : entry.getValue()) {
-            je.add(t.getMethod(mthd, null));

```

Comment:
`t.getMethod(mthd, null)` was just create a method handler, not exactly the method declaration, and noticed that the parameterTypes is `null`.
so the fix is call `JDTUtil.solveMethod()` to find the real method with parameterTypes
this fixed the issue that if people right click on the test method with @Parameter and @Optional parameters, the result view would display `null` rather than the parameter value